### DISCUSSION TITLE
绳网即将关闭

### DISCUSSION THREAD

**Discussion by share121** [share121/inter-knot#5495] [公告]
「绳网」论坛具有非常大的法律风险，所以我决定关停。

## 计划

基于绳网的原理，我可以开发出一款「个人博客」框架。

### 优势

1. 零编译时间，基于 Github Discussion，不需要任何编译时间
2. 部署方便，只要将仓库 fork 走，部署就完成了
3. 功能强大，Github Discussion 的所有功能都可以支持，如
   1. 🔍强大的搜索功能：Github Discussion 拥有丰富的搜索语法
   2. 分区功能：Github Discussion 中的 category 可以映射成不同的分区
   3. 协作功能：你可以在仓库设置中添加协作者，共同编辑博客

**Comment by Faethwith** [comment]
嗯呢呢（不是别关啊）

**Reply by Enigma-Soul** [reply]
自己做摆 Github Actions一键打包编译 有手就行 更别说我们这些干后端的

**Comment by 114514qwe-der** [comment]
不！！！！！
![img-206528119_1727360932059](https://github.com/user-attachments/assets/ebd6b1ad-465c-4ed7-b7f8-8bc2877d93ad)

**Comment by Jason555647** [comment]
不要不要😭

**Comment by Yizakl** [comment]
补药啊

**Comment by BJ-cmyk** [comment]
布豪！！！
![1727768562822](https://github.com/user-attachments/assets/f7c962a1-28d5-430c-9587-82afd2e25e65)

**Comment by hutao48** [comment]
_**不要啊**_
实在怕制裁可以禁止中国内地的IP访问啊
**所以能不能留下绳网**

**Comment by kunga555** [comment]
不不不不不

**Comment by wazds** [comment]
不要啊，求你别关了！

**Comment by jinlinahida** [comment]
补药呀，球球了

**Comment by Andrea0126** [comment]
啊好可惜啊😭
最近用的很開心的說
![img-1726458852112717d66365185ca81f71edb67b7008ce1480c995e395620f9ed85e9f6d14475e8](https://github.com/user-attachments/assets/d5a3c2e9-ecba-4591-bd53-63c8db1fd2cd)

**Comment by KELERSAMA** [comment]
😭回来吧我的绳网😭
🌟我最骄傲的信仰🌟
⚡️历历在目的教程⚡️
😭眼泪莫名在流淌😭
💥依稀记得钓鱼佬💥
👍还有给力的绳斤👍
⚡️把天马速递打退⚡️
✨通宵熬夜都不累✨

**Reply by liang129** [reply]
啊这

**Comment by cyq20091029** [comment]
不要啊，求求了<img src="https://upload-bbs.mihoyo.com/upload/2022/09/30/651fba00ba55004182fffbb5aa553c53_6086077287436441482.png" alt="米仔_膜拜" width="75" >

**Comment by CreamAlphaWolfram** [comment]
可以尝试维护（？）

**Comment by owofile** [comment]
站长幸苦了

**Comment by March-777th** [comment]
补药啊<img src="https://img-static.mihoyo.com/communityweb/upload/7bd889870907d2d96673d09afb53cf63.png" alt="三月七_哭" width="75" >

**Comment by xingyun-kb** [comment]
不是，啊？

**Comment by pinhexianran** [comment]
补药啊！！！！！

**Comment by Mrbeichen** [comment]
不要啊

**Comment by Bailinghai** [comment]
不要啊不要啊

**Comment by Doubudou45** [comment]
毁啦！都毁啦(╥ω╥`)

**Comment by xnpboke** [comment]
这下丸辣！

**Comment by imacatperson** [comment]
<img src="https://img-static.mihoyo.com/communityweb/upload/ee7bb800b0c90f66fa9e321cb6d8ee17.png" alt="抱大腿" width="75" ><img src="https://img-static.mihoyo.com/communityweb/upload/ee7bb800b0c90f66fa9e321cb6d8ee17.png" alt="抱大腿" width="75" ><img src="https://img-static.mihoyo.com/communityweb/upload/ee7bb800b0c90f66fa9e321cb6d8ee17.png" alt="抱大腿" width="75" ><img src="https://upload-bbs.miyoushe.com/upload/2023/01/17/004d0d701e82bdfaebfaabac327498da_131680735947960883.png" alt="求饶呜呜" width="75" ><img src="https://upload-bbs.miyoushe.com/upload/2023/01/17/004d0d701e82bdfaebfaabac327498da_131680735947960883.png" alt="求饶呜呜" width="75" ><img src="https://upload-bbs.miyoushe.com/upload/2023/01/17/004d0d701e82bdfaebfaabac327498da_131680735947960883.png" alt="求饶呜呜" width="75" >

**Comment by XiaoYumuamua** [comment]
不要啊😭😭😭😭😭

**Comment by muhan-972** [comment]
不要啊！！<img src="https://img-static.mihoyo.com/communityweb/upload/1642da910ce6a64b72eb436a008d3af8.png" alt="米游姬-求求你啦" width="75" ><img src="https://img-static.mihoyo.com/communityweb/upload/1642da910ce6a64b72eb436a008d3af8.png" alt="米游姬-求求你啦" width="75" >

**Comment by Aha-follower** [comment]
不要啊！

**Comment by qeryoupadhjlvnim** [comment]
别似啊

**Comment by RopeCraftsman** [comment]
不是？我9月31号来的，今天就被通知关了？

**Comment by build196** [comment]
<img src="https://img-static.mihoyo.com/communityweb/upload/ee7bb800b0c90f66fa9e321cb6d8ee17.png" alt="抱大腿" width="75" >

**Comment by SW-GFrog** [comment]
补药啊老大！没有绳网我可怎么活啊

**Comment by win98se** [comment]
不理解所谓“具有非常大的法律风险”是什么意思
在许许多多的国家，自制社交媒体是一点问题都没有的 (顶多就是做到大了，被政府看上了而已)，怎么到某个国家就有了？

**Reply by nostalgiatan** [reply]
应该是版权的问题吧，除此之外，也没有其他的问题了。

**Reply by nostalgiatan** [reply]
官方是没有回应的，有发过邮件。

**Reply by TNCNET** [reply]
主要是，这里毕竟可以被直接访问（我的所在地，我使用联通的网络就可以直连GitHub），访问难度很小。

而这里的话，没有屏蔽词等系统，并且总会有人发表确实违规违法的言论。额...

还有一个问题是，一旦出事没能及时监管，问题要找谁解决？站长？管理人员？GitHub？或者别的（）

总之确实很困难的说。

**Reply by TNCNET** [reply]
主要是，这里毕竟可以被直接访问（我的所在地，我使用联通的网络就可以直连GitHub），访问难度很小。

而这里的话，没有屏蔽词等系统，并且总会有人发表确实违规违法的言论。额...

还有一个问题是，一旦出事没能及时监管，问题要找谁解决？站长？管理人员？GitHub？或者别的（）

**Reply by TNCNET** [reply]
主要是，这里毕竟可以被直接访问（我的所在地，我使用联通的网络就可以直连GitHub），访问难度很小。

而这里的话，没有屏蔽词等系统，并且总会有人发表确实违规违法的言论。额...

还有一个问题是，一旦出事没能及时监管，问题要找谁解决？站长？管理人员？GitHub？或者别的（）

总之确实很困难的说。

**Reply by Blank0Page** [reply]
补药!

**Reply by EcoFur** [reply]
> 主要是，这里毕竟可以被直接访问（我的所在地，我使用联通的网络就可以直连GitHub），访问难度很小。
> 
> 而这里的话，没有屏蔽词等系统，并且总会有人发表确实违规违法的言论。额...
> 
> 还有一个问题是，一旦出事没能及时监管，问题要找谁解决？站长？管理人员？GitHub？或者别的（）
> 
> 总之确实很困难的说。

其实Ban掉国区IP就行了，或者部署到国外的云服务商也行

**Comment by OMO-114514** [comment]
虽然没用几天，但github做出来的绳网的确给了我一种普通人误入真绳网的感觉（我klf不懂英语）（哭笑

**Reply by Semcosm** [reply]
不不能没有绳网X﹏X@share121

**Comment by LDR-NGM** [comment]
AMNS,能留吗？这真的很酷很好玩欸！

**Comment by Liruijia666** [comment]
何也

**Comment by 123acnl** [comment]
补药

**Comment by NTTTU** [comment]
必然的

**Comment by xiaoliaoggg** [comment]
没有绳网，让我怎么活啊∑(;°Д°) ( ꒪Д꒪)ノ∑(O_O；)(((( ;°Д°))))

**Comment by Quatfolia** [comment]
不要啊，我才刚来… (*꒦ິ⌓꒦ີ)

**Comment by raoziqiang** [comment]
？、？

**Comment by Letyem** [comment]
——不，<img src="https://bbs-static.miyoushe.com/static/2023/05/23/a2fa805a0a782cdb8ab44ddaf8d5343d_962898009752203374.png" alt="阿姬倒地了04" width="75" >

**Comment by fzdqz** [comment]
不<img src="https://img-static.mihoyo.com/communityweb/upload/7bd889870907d2d96673d09afb53cf63.png" alt="三月七_哭" width="75" >

**Comment by StellaBigHero** [comment]
N.E.P.S.的人杀过来了？

**Comment by Loxia1121** [comment]
会在什么地方发佈「个人博客」的后续消息？在绳网？B站？

**Reply by Yizakl** [reply]
社区Discord服务器

**Reply by cyq20091029** [reply]
能给个服务器链接吗，APP上写的那个搜不到<img src="https://bbs-static.miyoushe.com/static/2023/05/23/a2fa805a0a782cdb8ab44ddaf8d5343d_962898009752203374.png" alt="阿姬倒地了04" width="75" >

**Comment by BaiHan2333** [comment]
要关闭了吗？好吧。有好多事情还没来得及跟大家分享

**Comment by wzyaeu** [comment]
啊？

**Comment by EcoFur** [comment]
从Star里面看到了这个，想看看绳网发展得什么样了，打开直接给我跳国家反诈了……然后就看到这个帖子了

**Reply by Yizakl** [reply]
大佬

**Reply by EcoFur** [reply]
> 大佬

我是菜13

**Reply by EcoFur** [reply]
> 大佬
预言家刀了：
https://github.com/share121/inter-knot/pull/823#issuecomment-2282788285

**Comment by rtcrtc123** [comment]
唉

**Comment by kanmanhin** [comment]
no！

**Comment by kanmanhin** [comment]
作者可以去mhy那边询问一下“重现绳网系统”是否能作为一个非盈利的免费二创继续存活呢？这样的话那就不用担心法律问题了吧

**Reply by Yizakl** [reply]
曾经有给米哈游发过邮件
但是米哈游似乎没有回信

**Reply by EcoFur** [reply]
> 作者可以去mhy那边询问一下“重现绳网系统”是否能作为一个非盈利的免费二创继续存活呢？这样的话那就不用担心法律问题了吧

不是盈利的问题，是社区的问题，这里相当于一个无法被中国政府监管的论坛，可能会有反动、涉黄、违法内容。

**Comment by LingHe9639** [comment]
啊？不是，别啊

**Comment by LingHe9639** [comment]
啊？不是，别啊

**Comment by c328422** [comment]
所以可以理解是因為中華人民共和國的法律問題，此社區可以接觸國家安全，這麼理解沒啜吧？

**Comment by c328422** [comment]
感覺可以將繩網的管理丟給願意接手的非中華人民共和國管轄範圍的人處理，這樣是否能讓繩網活下去？

**Comment by xingyuehao** [comment]
那新绳网建成后我们怎么获得相关信息呢？DC官群那个链接挂vpn好像也打不开，其他的邀请链接也说过期了。有没有人捞捞我啊（悲）

**Comment by yuxwd** [comment]
快端上来吧！！！！
wc一键部署

**Comment by c328422** [comment]
繩網有人接了？怎麼突然可以連了？

**Reply by kanmanhin** [reply]
并没有

**Comment by EcoFur** [comment]
终于有新Discussion了


Sent from my Apple Watch

**Comment by blqdswkl** [comment]
2024/11/29
我说邮箱这两月怎么这么安静

**Comment by EcoFur** [comment]
iCloud终于有消息了，学校里就只能看看iCloud和iMassage 


Sent from my Apple Watch

**Comment by JT9364** [comment]
辛苦了

**Comment by BigShen5233** [comment]
不要啊

**Comment by mythca** [comment]
补药啊！
